lucuma-schemas/src/main/scala/lucuma/schemas/odb/ExecutionVisitsSubquery.scala

```diff
@@ -9,16 +9,30 @@ import lucuma.schemas.decoders.given
 import lucuma.schemas.model.ExecutionVisits
 
 object ExecutionVisitsSubquery
-    extends GraphQLSubquery.Typed[ObservationDB, ExecutionVisits]("Execution"):
+    extends GraphQLSubquery.Typed[ObservationDB, Option[ExecutionVisits]]("Execution"):
   override val subquery: String = s"""
     {
       config {
         instrument
-        ... on GmosNorthExecutionConfig {
-          ...gmosNorthStaticConfigFields
+        gmosNorth {
+          static {
```

Comment:
I guess we lost the ability to abstract the common fields away to a `fragment`. The implementations for each instrument are now unrelated, right?

Reply:
The fragments for each instrument were separate, anyway. However, I `inlined` these 2 fragments because

1. Each one was only used once, anyway.
2. This avoids having to repeat those fragments at the use site of the subquery.